src/AppInstallerRepositoryCore/Microsoft/ARPHelper.h

```diff
@@ -59,7 +59,7 @@ namespace AppInstaller::Repository::Microsoft
         // The priority is:
         //  DisplayVersion
         //  Version
-        //  MajorVerison, MinorVersion
+        //  MajorVersion, MinorVersion

```

Comment:
Google Sheets suggested `Verizon`

Reply:
Major Verizon

A lot of these changes are my extremely common typos that I failed to correct.  I'm sure that some sort of ML (or just analysis of qwerty) could probably figure out that one of my fingers jumps the gun on the others occasionally.